README.md

```diff
@@ -0,0 +1,21 @@
+# Cruise ships project
+This project is part of the Manchester Codes programming fundamentals module. 
+
+## Description 
+This project is part of the Manchester Codes programming fundamentals module.
+
```

Comment:
Could some installation and setup instructions be included here? Also how are the methods used?